glossary/he/index.md

```diff
@@ -95,4 +95,4 @@ Information Asset Register רשימות IAR מיועדות לאסוף ולארג
 
 Open Government Data :מונח:’מידע פתוח הנוצר על-ידי הממשלה.’ זהו מידע הנוצר במהלך הפעילות השגרתית של משרדי הממשלה ,מבלי שהוא חושף יחידים או מידע מסחרי רגיש. מידע ממשלתי פתוח הינו תת-מערך של :מונח:’מידע מהסקטור הציבורי’, שהינו רחב יותר. לפרטים, ראו <http://opengovernmentdata.org>.
 
-Tab-separated values ערכים מופרדי טאב (TSV), הם צורה מקובלת של פורמט קבצי טקסט לשיתוף מידע טבלאי. פורמט זה פשוט מאוד ו:מונח:’קריא על-ידי מחשב.
```

Comment:
@jgkim - Hebrew is my Native language, this is why I got confused. Understood now.